check-tables.sql

```diff
@@ -40,6 +40,9 @@
 --     ('new user','new comment',9)  RETURNING *;
 -- -- SET foreign_key_checks = 1;
 
-SELECT COUNT (comments.article_id) AS comment_count, articles.title, articles.topic, articles.author, articles.created_at, articles.votes,articles.article_img_url, comments.article_id FROM articles JOIN comments ON articles.article_id = comments.article_id
-WHERE topic = 'mitch'
-ORDER BY created_at DESC
\ No newline at end of file
+-- SELECT COUNT (comments.article_id) AS comment_count, articles.title, articles.topic, articles.author, articles.created_at, articles.votes,articles.article_img_url, comments.article_id FROM articles JOIN comments ON articles.article_id = comments.article_id
+-- WHERE topic = 'mitch'
+-- GROUP BY articles.title, articles.topic, articles.author, articles.created_at, articles.votes,articles.article_img_url, comments.article_id
+-- ORDER BY created_at DESC
```

Comment:
Remember to delete commented out code - we want to keep things clean, so that the main branch isn't cluttered